metadata-models/src/main/pegasus/com/linkedin/datajob/DataJobInfo.pdl

```diff
@@ -35,7 +35,7 @@ record DataJobInfo includes CustomProperties, ExternalReference {
   /**
    * Datajob type
    */
-  type: union[AzkabanJobType]
```

Comment:
should we add a note that `AzkabanJobType` is deprecated?

Reply:
Yes!